appveyor.yml

```diff
@@ -7,37 +7,11 @@ environment:
       EXAMPLE: 01-executable
       CONFIG: Release
 
-    - TOOLCHAIN: "ninja-vs-12-2013-win64"
+    - TOOLCHAIN: "mingw-cxx17"
       EXAMPLE: 01-executable
       CONFIG: Release
-
-    - TOOLCHAIN: "nmake-vs-12-2013-win64"
```

Comment:
Please keep all other toolchains. You can just substitute `mingw` with `mingw-cxx17`.